### PR TITLE
testing/stress-ng: New aport

### PR DESCRIPTION
**Pull request by oliv3r**
Signed-off-by: Olliver Schinagl <oliver@schinagl.nl>

### CODE REVIEW

File: testing/stress-ng/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	libgcrypt-dev
+	linux-headers
+	lksctp-tools-dev
+	zlib-dev
+"
+options="!check" # Tests are unreliable with qemu and take a long time
+subpackages="${pkgname}-doc ${pkgname}-examples"
+source="
+	https://kernel.ubuntu.com/~cking/tarballs/${pkgname}/${pkgname}-0.09.54.tar.xz
+
```
Comment (Ikke) [inline testing/stress-ng/APKBUILD]: redundant blank line

File: testing/stress-ng/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+doc()
+{
+	mkdir -p "${subpkgdir}/usr/share/man/"
+	mv "${pkgdir}/usr/share/man/man1/" "${subpkgdir}/usr/share/man/"
+	rm -r "${pkgdir}/usr/share/man/"
+	rmdir "${pkgdir}/usr/share" || true
+}
+
+examples()
```
Comment (Ikke) [inline testing/stress-ng/APKBUILD]: What is the reason you opted for a separate examples subpackage?

File: testing/stress-ng/APKBUILD (hunk 1 above)
Comment (oliv3r) [reply testing/stress-ng/APKBUILD]: actually, it is the separator between the source and the patch. I saw a lot of packages making that distinction which, for readability, makes sense.

File: testing/stress-ng/APKBUILD (hunk 2 above)
Comment (oliv3r) [reply testing/stress-ng/APKBUILD]: Also, I noticed some packages doing that. And in the alpine sense, it makes sense to do so. The goal is to have a minimal amount of packages/clutter installed. For containers/embedded, I totally see that users want only the package. For a desktop (where space isn't a concern) having all the docs is not too bad. The examples you really only ever want is when you are experimenting. So we can drop it all together of course, but I figured, why not?

**Pull request closed without merging**